demos/projects/PC/linux/tests/aux_needed_functions.c

```diff
@@ -0,0 +1,256 @@
+/* Standard includes. */
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <time.h>
+#include <strings.h>
+#include <unistd.h>
+#include <assert.h>
+
+/* FreeRTOS includes. */
+#include <FreeRTOS.h>
+#include "task.h"
+
+/* TCP/IP stack includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_Sockets.h"
+
+#include "mbedtls/entropy.h"
+
+#define mainHOST_NAME           "RTOSDemo"
+#define mainDEVICE_NICK_NAME    "linux_demo"
+
+/*
+ * Just seeds the simple pseudo random number generator.
+ *
+ * !!! NOTE !!!
+ * This is not a secure method of generating random numbers and production
+ * devices should use a true random number generator (TRNG).
+ */
+static void prvSRand( UBaseType_t ulSeed );
+
+/* The default IP and MAC address used by the demo.  The address configuration
+ * defined here will be used if ipconfigUSE_DHCP is 0, or if ipconfigUSE_DHCP is
+ * 1 but a DHCP server could not be contacted.  See the online documentation for
+ * more information. */
+static const uint8_t ucIPAddress[ 4 ] = { configIP_ADDR0, configIP_ADDR1, configIP_ADDR2, configIP_ADDR3 };
```

Comment:
If this is just for compiling, do we need this?  It's not referenced and it's a static variable.  General question for other static only stuff that's not called inside this file.

Reply:
removed everything that isn't needed.